modules/account-lib/test/unit/coin/avax/keyPair.ts

```diff
@@ -0,0 +1,167 @@
+import should from 'should';
+import { KeyPair } from '../../../../src/coin/avax/keyPair';
+import { TEST_ACCOUNT } from '../../../resources/avax/avax';
+
+describe('Avax Key Pair', () => {
+  describe('should create a valid KeyPair', () => {
+    it('from an empty value', () => {
+      const keyPairObj = new KeyPair();
+      const keys = keyPairObj.getKeys();
+      should.exists(keys.prv);
+      should.exists(keys.pub);
+      should.equal(keys.prv!.length, 64);
+      should.equal(keys.pub.length, 130);
```

Comment:
I'm a little confused here, I used avalanche-sdk's `getPublicKeyString()` on a key pair, and I got a public key with 66 chars (`8jAVW8gn6WaARg4PUPdbpnenqNYBMtUWSzvoh2uVgU91T1LfdL`)

Reply:
The reason for that is that `avalanche-sdk ` generates keys in Avax Native Format. i.e, P|X|C - <66 base58 encoded string>
However, that C-<key> is only valid when importing and exporting balance between chains. 
When we use the C-Chain it works as Ethereum EVM so the accepted format for keys is the same as ethereum (130 uncompressed public key)

I originally had include methods to convert from EVM to AVM keys formats, but avalanche-sdk requires Node >= v14.17
So I removed that since we still need to keep giving support to Node 12 the next 7 months I think.

Reply:
I see. `getKeys()` returns pub in compressed format when the keypair is created from an `xprv`, but it returns an uncompressed pub if a `prv` is used.
@BitGo/eth-team is that behavior intentional? @smaria95's change makes it consistently return pubs in uncompressed format. Would that cause an issue when we create a controller in wp that extends from `AbstractEthLikeCoin`?

Reply:
@smaria95 and I looked for call sites to this logic from SDK, Wallet-platform and OVC. Nothing appears to be affected by this change. 
At some point we could change the ETH KeyPaid implementation to match this one for consistency.